File: OCP/MungedSolution/MarsRoverKata/Obstacle.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MarsRoverKata
{
    public class Obstacle
    {
        public Point Location { get; set; }

        public Obstacle(Point location)
        {
            Location = location;
        }

        public bool IsDestructable { get { return true; } }
    }
}

File: OCP/Java_Final/MarsRoverKata/src/main/java/MarsRoverKata/Obstacle.java
package MarsRoverKata;

public class Obstacle {
    private Point location;

    protected void setLocation(Point location) {
        this.location = location;
    }

    public Point getLocation() {
        return location;
    }

    public Obstacle(Point location) {
        this.location = location;
    }

    public boolean isDestructable() {
        return true;
    }
}
